itest/rust/src/object_tests/dyn_gd_test.rs

```diff
@@ -515,7 +515,7 @@ struct RefcDynGdExporter {
     #[var]
     first: Option<DynGd<Object, dyn Health>>,
     #[export]
-    second: Option<DynGd<foreign::NodeHealth, dyn InstanceIdProvider<Id = InstanceId>>>,
+    second: Option<DynGd<Node, dyn InstanceIdProvider<Id = InstanceId>>>,
```

Comment:
With concrete types being allowed again, this could again be `foreign::NodeHealth`.

Maybe add a comment before `#[export]`:
```rs
// Using DynGd with concrete type foreign::NodeHealth doesn't give benefits over Gd, but is allowed.
```